### PR TITLE
cleanup fuzzy

### PR DESCRIPTION
**Pull request by ReenExe**

### CODE REVIEW

**Comment by ReenExe** [comment]
@ruflin, covers after #1125

File: test/lib/Elastica/Test/Query/FuzzyTest.php [hunk 1]
```diff
@@ -100,20 +111,32 @@ public function testQuery()
...
-        $this->setExpectedException('Elastica\Exception\InvalidException');
+    /**
+     * @group unit
+     */
+    public function testResetSingleField()
+    {
         $query = new Fuzzy();
-        $query->setField('name', []);
+        $query->setField('name', 'value');
+        $query->setField('name', 'value1');
```
Comment (ruflin) [inline test/lib/Elastica/Test/Query/FuzzyTest.php]: An assertion should be added here to check if the field is resetted.

**Comment by ruflin** [comment]
LGTM. Left one minor comment. Thanks for adding the tests.

**Comment by ruflin** [comment]
Merged. Thanks.

**Pull request merged**